server/static/js/index/home_page_processing.js

```diff
@@ -0,0 +1,122 @@
+"use strict";
+const TABLE_USERS = '#table_users';
+const TABLE_DEVICES = '#table_device';
+let renderTableDelay = 200;
+
+// Callback function, called on success, for reading device data
+function drawTableDevices (data) {
```

Comment:
Just a minor correction here - drawDevicesTable. A bit of english grammar to smooth the edges ^_^